src/components/UserDropdown/index.tsx

```
@@ -1,166 +1,228 @@
+            </span>
+          </DropdownMenuItem>
+
+          {filteredMenuItems.map((item, index) => {
+            // Skip the profile item as we've added it manually at the top
+            if (item.label === "Hồ sơ cá nhân") return null;
+
+            const Icon = item.icon;
+            const isStaffOnly =
+              item.roles &&
+              item.roles?.[0]?.includes("Staff") &&
+              !item.roles?.[0]?.includes("Guest");
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)